content/src/GlueTemplate.fs

```diff
@@ -1,5 +1,14 @@
-module GlueTemplate.Exports
```

Comment:
The name `Exports` was made neutral on purpose and this is a common pattern for bindings at least.